openai-proxy/py_proxy/proxy.py

```diff
@@ -174,9 +174,9 @@ def opensearch_version(retries=3):
         return response.json()['version']['number'], 200
     except Exception as e:
         if retries <= 0:
-            print(f"OpenSearch not standing at {OPENSEARCH_URL}. Out of retries.")
+            logging.error(f"OpenSearch not standing at {OPENSEARCH_URL}. Out of retries. Final error {e}")
```

Comment:
This is basically a way to go from stdout to stderr, right?  I assume it'll prepend with 'ERROR:root:'.  Do you know how to change 'root' to something more meaningful?  I could use it too.

Reply:
It does two things: 1) write to stderr (mostly irrelevant since it's coming out via logs), and 2) flush after writing (very important, otherwise you don't see the messages at all)

Eventually we want to change the user away from root. You can configure the format as is done in:
https://github.com/aryn-ai/managed-service/pull/39/files#diff-5fd88c4e154ad2946ef2d18f927250b680c86a7abd44fcc80deebdac016b0bfc